src/App.tsx

```diff
@@ -9,7 +9,7 @@ function App() {
       return
     }
 
-    window.StockTicker.addListener((tick) => {
+    window.StockTicker.addListener((tick: any) => {
```

Comment:
I made this `any` as a way for us to test devs on whether they will make their own types or not, if that's not something we want, I can make the type and reference it here

Reply:
I think for the most part they will probably leave it as `any` but we could also ask them to make it a Type. I think it is fine as `any`

Reply:
lulz... _classic_ typescript... YES, I'M A H8R!